SPEC.md

```diff
@@ -73,7 +73,15 @@ The fuzz tests aim to buttress the message conformance tests with extra verifica
 
 # Test Index
 
-The test index makes use of symbolic language in describing connection and message sending directions. As a convention, Ziggurat test nodes are to the left of the connection/message arrows, and Rippled instances are to the right: `A -> B` and `A <- B`. In this way, `->` signifies "Ziggurat connects to Rippled" and `<-` signifies the opposite. Furthermore, `-> ping` signifies "Ziggurat sends a `Ping` message to Rippled" and `<- pong` signifies "Rippled sends a `Pong` message to Ziggurat". Lastly, `<>` signifies a completed handshake, in either direction.
+The test index makes use of symbolic language in describing connection and message sending directions.
+
+| Symbol | Meaning |
```

Comment:
Nice 👍

Reply:
Let's eventually align all three repos with the same table